control-plane/pkg/reconciler/trigger/trigger_test.go

```diff
@@ -2825,6 +2827,85 @@ func triggerFinalizer(t *testing.T, format string, env config.Env) {
 				},
 			},
 		},
+		{
+			Name: "Reconciled normal - With Subscriber CA Cert",
```

Comment:
Can we add another similar test for the dead letter sink CA certs? or even expand this one covering dead letter sink CA certs?